frontend/src/views/account-mgmt/ManageOrganizationView.vue

```diff
@@ -84,7 +94,11 @@ export default {
       panelYourFacilities: [0],
       panelOtherFacilities: [0],
       loading: false,
+      loadingInclusionPolicy: false,
       organization: undefined,
+      uploadedDocuments: [],
+      documentsToDelete: [],
+      documentsToUpload: [],
```

Comment:
I don't think these need to be in the data. They're only used within Save. You can just pass them around as parameters.

Reply:
Fixed.